common/Kconfig

```diff
@@ -937,7 +937,7 @@ config HASH
 
 config AVB_VERIFY
 	bool "Build Android Verified Boot operations"
-	depends on LIBAVB && FASTBOOT
```

Comment:
We can drop this patch as we will not be using avb anymore.

Reply:
sure. I thought we needed avb for the provisioning of the keys (although now that I remember we are only going to be provisioning from linux) ok